### DISCUSSION TITLE
delete specific emails

### DISCUSSION THREAD

**Discussion by azc666** [beyondcode/herd-community#418] [Q&A] [answered]
Just activated pro. mail works great but I don't see a way to delete a specific email just the mailbox. can I do that? I would like to keep specific emails and don't want to delete the mailbox. Thx.

**Comment by mpociot** [comment, marked as answer]
Just press backspace when a mail is selected 🙂

**Comment by azc666** [comment]
Thx for quick response!!!

The one thing I never tried. my bad.

👍🏻

Allen
--------------------------------
Allen Cohen, President
Graphics + Design
Carter Printing Company
Certified Apple Consultant
Microsoft Partner
1111 W. Cass Street
Tampa, FL 33606
813-254-9444
***@***.***
www.g-d.com
